backend/Data/PIMSContextFactory.cs

```diff
@@ -11,16 +11,17 @@ public class PIMSContextFactory : IDesignTimeDbContextFactory<PIMSContext>
     {
         public PIMSContext CreateDbContext(string[] args)
         {
+            DotNetEnv.Env.Load();
             string environment = Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT");
 
             var config = new ConfigurationBuilder()
                 .SetBasePath(Directory.GetCurrentDirectory())
                 .AddJsonFile("appsettings.json", optional: true, reloadOnChange: true)
                 .AddJsonFile($"appsettings.{environment}.json", optional: true, reloadOnChange: true)
+                .AddEnvironmentVariables()
                 .AddJsonFile("connectionstrings.json", optional: true, reloadOnChange: true)
```

Comment:
You can now have configuration for the `dotnet-ef` CLI that doesn't interfere with your `.env` and docker containers.